CoroutineTemplate/app/src/main/java/co/nimblehq/coroutine/ui/screens/compose/composables/ComposeScreen.kt

```diff
@@ -0,0 +1,48 @@
+package co.nimblehq.coroutine.ui.screens.compose.composables
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.material.CircularProgressIndicator
+import androidx.compose.material.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import co.nimblehq.coroutine.domain.data.entity.UserEntity
+import co.nimblehq.coroutine.ui.screens.compose.theme.ComposeTheme
+
+@Suppress("LongMethod", "LongParameterList")
+@ExperimentalComposeUiApi
+@Composable
+fun ComposeScreen(
+    users: List<UserEntity>,
+    showLoading: Boolean,
+    textFieldValue: String,
+    onTextFieldValueChange: (String) -> Unit,
+    onUserItemClick: (String) -> Unit
+) {
+    ComposeTheme {
+        Scaffold(
+            topBar = {
+                TitleBar(
+                    title = "Jetpack Compose",
+                    textFieldValue = textFieldValue,
+                    onTextFieldValueChange = onTextFieldValueChange
+                )
+            }
```

Comment:
we should only use the `topBar` widget if there is no action bar on the top. 😄 
![image](https://user-images.githubusercontent.com/60863885/126962994-c3667e05-5756-4d00-87a6-beb20921ba61.png)
You may consider moving the text field to the body of the `Scaffold` if we need to use this text field. 🙏

Reply:
What do you think if we using `Theme.AppCompat.Light.NoActionBar` in `styles.xml` to hide the above action bar instead? 😄

Reply:
@chornerman that works for me 🥇

Reply:
Resolved 🙏🏼